packages/v4/src/content/design-guidelines/components/card/card.md

```diff
@@ -3,116 +3,99 @@ id: Card
+## Usage
+Use cards:
+- To visually group content or distinguish information from the rest of a page’s content. Examples include dashboards and catalog views.
+- To organize content in layouts with multiple columns or grids.  
+- To compare separate groups of information.  
+- To make media easier to scan, such as images or videos.  
 
-<img src="./img/inline-actions.png" alt="cards with inline actions" /> 
+**Never** use a card to show a dense amount of repetitive, related content. Use a [data list](/components/data-list) or [table](/components/table) view instead.
 
-#### Multiple inline actions
-If multiple actions can be performed on a single card, place those actions in a kebab. Position the kebab in the top-right corner of the header.
+Cards primarily appear in [catalog views](#cards-in-catalog-views), [primary-detail views](#cards-in-primary-detail-views), and [dashboards](#cards-in-dashboards).
 
-<img src="./img/multiple-inline.png" alt="cards with multiple inline actions in a kebab" /> 
+### Cards in catalog views
 
-#### Global actions 
-If an action can be performed globally on multiple cards, or if it leads to multiple pieces of content at the same time, place the action in a toolbar above the card view. In the top-right corner of your card, add a checkbox so that users can select one or more items.
+Use cards in a catalog view to present users with a series of selectable items, typically arranged into a grid.
 
-<img src="./img/global-actions.png" alt="card view with global action toolbar" /> 
+Note that this view uses a multi-select card so that bulk actions can be performed from the toolbar on a group of cards at once. For more information on cards used in a catalog or card view, see the [card view design guidelines](/demos/card-view/design-guidelines).
 
-1. Upon hover, the card’s shadow expands.
-2. Global actions are placed in the toolbar.
-3. The card’s checkbox allows a user to perform an action on multiple cards. When a user selects a card, the checkbox turns blue to indicate selection.
+<img src="./img/multi-select-cards.png" alt="cards in a catalog view " width="1210" />
 
-#### Action card
-You may use an extra-small empty state inside of a card to present the user with another means of adding a new card/item to their card view. If used, we recommend using it **in addition to** a primary button in a toolbar, to ensure that the user is still able to perform the action no matter what page of the card view they are on. The placement of the action card should be consistent with where a new card will be added, most likely the very first or the very last card in the card view.
 
-<img src="./img/add-card-empty-state.png" alt="card view with addition card" /> 
+### Cards in primary-detail views
 
+Use selectable cards when you want to present an overview of a set of objects and then examine the details. Here, a details of different projects can be displayed in a side-by-side primary-detail view.
```

Comment:
Would rephrase to remove the "a" in "Here, **a** details of different..." Somehow reads weird to be otherwise?